ksql-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -202,6 +202,24 @@
   public static final String KSQL_AUTH_CACHE_MAX_ENTRIES_DOC = "Controls the size of the cache "
       + "to a maximum number of KSQL authorization responses entries.";
 
+  public static final String KSQL_INTERNAL_HIDDEN_TOPICS_CONFIG = "ksql.internal.hidden.topics";
+  public static final String KSQL_INTERNAL_HIDDEN_TOPICS_DEFAULT = "_confluent.*,__confluent.*"
+      + ",_schemas,__consumer_offsets,__transaction_state,connect-configs,connect-offsets,"
+      + "connect-status,connect-statuses";
+  public static final String KSQL_INTERNAL_HIDDEN_TOPICS_DOC = "List of topics that will not be "
+      + "visible when running the SHOW TOPICS command unless SHOW ALL TOPICS is used. This list "
+      + "is comma separated and may use Java regular expressions to specify each topic (i.e. "
+      + " _confluent.* accepts any topic that starts with the _confluent prefix).";
+
+  public static final String KSQL_INTERNAL_READONLY_TOPICS_CONFIG = "ksql.internal.readonly.topics";
```

Comment:
Inline with the config for hidden topics; I think the following is clearer:

```suggestion
  public static final String KSQL_READONLY_TOPICS_CONFIG = "ksql.readonly.topics";
```